firestore/objc/firestore-smoketest-objc/ViewController.m

```diff
@@ -1200,4 +1200,95 @@ - (void)countAggregateQuery {
     // [END count_aggregate_query]
 }
 
+- (void)sumAggregateCollection {
+  // [START sum_aggregate_collection]
+  FIRQuery *query = [self.db collectionWithPath:@"cities"];
+  FIRAggregateQuery *aggregateQuery = [query aggregate:@[
+    [FIRAggregateField aggregateFieldForSumOfField:@"population"]]];
+  [aggregateQuery aggregationWithSource:FIRAggregateSourceServer
+                             completion:^(FIRAggregateQuerySnapshot *snapshot,
+                                          NSError *error) {
+    if (error != nil) {
+      NSLog(@"Error fetching aggregate: %@", error);
+    } else {
+      NSLog(@"Sum: %@", [snapshot valueForAggregateField:[FIRAggregateField aggregateFieldForSumOfField:@"population"]]);
+    }
+  }];
+  // [END sum_aggregate_collection]
+}
```

Comment:
nit: use 4 space indents to be consistent with the rest of the file

Reply:
Done. Thanks.